README.rst

```diff
@@ -1,151 +1,176 @@
+
+
+Enabling the XBlock in a course
+*******************************
+
+When the XBlock has been installed, you can enable it in a course from Studio
+through the **Advanced Settings**.
 
-Internationalization (i18n) is when a program is made aware of multiple languages.
-Localization (l10n) is adapting a program to local language and cultural habits.
+1. Go to Studio and open the course you want to add the XBlock to.
+2. Go to **Settings** > **Advanced Settings** from the top menu.
+3. Search for **Advanced Module List** and add ``"filesmanager"`` to the list.
+4. Click **Save Changes** button.
 
-Use the locale directory to provide internationalized strings for your XBlock project.
-For more information on how to enable translations, visit the
-`Open edX XBlock tutorial on Internationalization <https://edx.readthedocs.org/projects/xblock-tutorial/en/latest/edx_platform/edx_lms.html>`_.
+   .. image:: https://github.com/eduNEXT/xblock-filesmanager/assets/64033729/5f7c99b8-31d2-492f-8573-88ae8748166b
+      :alt: Enable XBlock in a course
 
-This cookiecutter template uses `django-statici18n <https://django-statici18n.readthedocs.io/en/latest/>`_
-to provide translations to static javascript using ``gettext``.
 
-The included Makefile contains targets for extracting, compiling and validating translatable strings.
-The general steps to provide multilingual messages for a Python program (or an XBlock) are:
+Adding a Files Manager Component to a course unit
+*************************************************
 
-1. Mark translatable strings.
-2. Run i18n tools to create raw message catalogs.
-3. Create language specific translations for each message in the catalogs.
-4. Use ``gettext`` to translate strings.
+From Studio, you can add the Files Manager Component to a course unit.
 
-1. Mark translatable strings
-=============================
+1. Click on the **Advanced** button in **Add New Component**.
 
-Mark translatable strings in python::
+   .. image:: https://github.com/eduNEXT/xblock-filesmanager/assets/64033729/7c4cfde4-f2b2-4334-b646-c302dea9c515
+      :alt: Open Advanced Components
 
+2. Select **filesmanager** from the list.
 
-    from django.utils.translation import ugettext as _
+   .. image:: https://github.com/eduNEXT/xblock-filesmanager/assets/64033729/d5a524e5-ce0e-4cec-a336-3b04737fb373
+      :alt: Select Files Manager Component
 
-    # Translators: This comment will appear in the `.po` file.
-    message = _("This will be marked.")
 
-See `edx-developer-guide <https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/internationalization/i18n.html#python-source-code>`__
-for more information.
+Using the Files Manager Component
+*********************************
 
-You can also use ``gettext`` to mark strings in javascript::
+1. Go to edit section of the component from Studio.
+2. Create your folders structure by clicking on the **Create folder** button.
+3. Upload files by clicking on the **Upload files** button.
```

Comment:
can we specify that the files uploaded by the xblock are also added to the course assets (so they can be viewed from `Content > Files`)

Reply:
Done